modelforge/potential/parameters.py

```diff
@@ -136,10 +130,22 @@ class PerAtomEnergy(ParametersBase):
     keep_per_atom_property: bool = False
 
 
+class CoulomPotential(ParametersBase):
+    electrostatic_strategy: str = "coulomb"
+    maximum_interaction_radius: Union[str, unit.Quantity]
+    from_atom_to_molecule_reduction: bool = False
+    keep_per_atom_property: bool = False
+
+    converted_units = field_validator(
+        "maximum_interaction_radius",
```

Comment:
I think this should be the maximum_coulomb_interaction_radius (added in after the multiple pair listsPR). This might need some extra logic I. There that states like “use_local_interaction_pairlist” if cutoffs match.

Reply:
thanks! I changed the variable name